docs/assets/css/site.css

```diff
@@ -0,0 +1,520 @@
+/*
+    General Site Styling
+*/
+
+body {
+    background-color: #000;
+}
+
+h3 {
+    color: #fff;
+}
+
+p {
+    color: #fff;
+}
+
+a {
+    color: #b8ffd8;
+    transition: color 0.3s ease-in-out;
+}
+
+a:hover,
+a:visited {
+    color: #000;
+}
+
+.container.m-t-xl.m-b-xl {
+	background-color: #e5e5e5;
+    color: #fff;
+}
+
+/*
+    Page Layout
+*/
+
+.wrapper-content {
+    padding: 35px 20px 40px;
+}
+
+.sidebar {
+    top: 114px;
+}
+
+/*
+    Top Navigation Bar
+*/
+
+.top-header {
+    background-color: #0a0a0a;
+    height: 54px;
+    border-bottom: 2px solid #b8ffd8;
+}
+
+.top-header.base-shadow {
+    box-shadow: 0px 2px 5px 0px #000000;
+}
+
+.top-header .container .logo {
+    height: 52px;
+}
+
+.top-header .container .logo .image {
+    max-height: 30px;
+}
+
+.top-header .container .menu .menu-item {
+    padding: 4px 20px;
+    color: #b8ffd8;
+}
+
+.top-header .container .menu .menu-item.active {
+    background-color: #b8ffd8;
```

Comment:
Active menu item text is slightly hard to read:

![image](https://i.gyazo.com/6bbbef8b1f0253c40286968b054b9231.gif)

Perhaps set the text color to the current hover text color, I notice most other CAF docs sites dont text to change text color when hovering on the active menu item so perhaps this will solve this.